config/stats.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+import (
+	"github.com/qri-io/jsonschema"
+)
+
+// Stats configures a qri stats
+type Stats struct {
```

Comment:
There's some confusion here between **stats** configuration and **stats cache** configuration, partially because we didn't talk about this at great length 😄. 

We need a place to configure how stats are calculated. As an example, a user may be working with massive datasets on a slow machine, and want to lower `stats.StopFreqCountThreshold` to `1` so that frequency calculation is skipped entirely, regardless of where it's being cached.

So we need to be able to configure stats, and move the conversation about cache configuration into it's own structure. This is a great use case for a struct literal:

```go
// Stats configures qri statistical metadata calculation
type Stats struct {
  Cache struct {
    Type string `sson:"type"`
    MaxSize  uint64 `json:"maxsize"`
    Path  string `json:"path,omitempty"`
  }
  // For later addition:
  // StopFreqCountThreshold int
}
```